src/win/fs-event.c

```diff
@@ -455,36 +460,34 @@ void uv_process_fs_event_req(uv_loop_t* loop, uv_req_t* req,
             sizew = -1;
           }
 
-          if (filenamew) {
-            /* Convert the filename to utf8. */
+          /* Convert the filename to utf8. */
+          size = WideCharToMultiByte(CP_UTF8,
```

Comment:
My comment came in at the same time as @saghul's. Mine was a response to @bnoordhuis.